src/util/useId.ts

```diff
@@ -12,7 +12,7 @@ export const useId =
   function useId() {
     if (process.env.NODE_ENV !== 'production' && showWarning) {
       console.warn(
-        'EDS useId is not ssr friendly if using React <18 as it uses the native React.useId hook if available https://react.dev/reference/react/useId.',
+        "React < 18: EDS's `useId` is not SSR friendly. We recommend applying an ID package like `uniqueId` or using React 18's built-in `React.useId` https://react.dev/reference/react/useId",
```

Comment:
1) Is there a way this warning could talk about the "how". For example, if I see this message how would I apply uniqueId to resolve the warning?

2) Will this message ever show if using React >= 18? The message seem's to indicate some opt-in behavior

Reply:
@jeremiah-clothier 

1. that's fair. I didn't want to go into too much detail, as this would have only affected along, and they just moved IDs onto `uniqueId`. I can rephrase to say someone should import and use uniqueId or similar to set the ID prop for safety
2. once they move to React 18, they can keep using `uniqueId` OR, allow this to fall back to react's SSR-safe `.useId()`. This message only appears when someone is using React <= 17 AND isn't setting IDs on form fields themselves.

For context, this naive implementation of an ID generator can lead to race conditions in rare cases, or cause different IDs to appear when rendered on the server vs the client browser.